File: 프로그래머스/0/120852. 소인수분해/소인수분해.js
function solution(n) {
    const answer = [];
    
    while(n !== 1) {
        for (let i = 2; i < n + 1; i++) {
            if (n % i === 0) {
                if (!answer.includes(i)) answer.push(i);
                n /= i;
                break;
            }
        }
    }
    
    return answer.sort((a, b) => a - b);
}

File: 프로그래머스/0/120852. 소인수분해/소인수분해.py
def solution(n):
    answer = []
    
    while (n != 1) :
        for i in range(2, n+1):
            if (n % i == 0) :
                if (i not in answer):
                    answer.append(i)
                n = n // i
                break
    answer.sort()     
    return answer